src/index.js

```diff
@@ -85,10 +83,12 @@ export class RootComponent extends React.Component {
 		return (
 			<Editor
 				initialHtml={ initialData }
-				initialHtmlModeEnabled={ initialHtmlModeEnabled }
+				initialHtmlModeEnabled={ this.props.initialHtmlModeEnabled }
```

Comment:
Not suggesting a change here, just curious if there is a rationale for why `initialHtmlModeEnabled` was removed from a destructuring while the other three props were added to it?

Reply:
I think this came about as just a resolution to a merge conflict. For the other const declarations it was suggested to use this.props.whatever which I had converted to on a previous merge then I think just ran into issues when resolving it later